azure-mgmt-batch/src/main/java/com/microsoft/azure/management/batch/implementation/BatchAccountsImpl.java

```diff
@@ -66,6 +66,9 @@ protected BatchAccountImpl wrapModel(String name) {
 
     @Override
     protected BatchAccountImpl wrapModel(BatchAccountInner inner) {
+        if (inner == null) {
```

Comment:
might be simpler/cleaner to just say:
return (inner == null) ? null : new .... ;